### PR TITLE
Skip flaky story analytics integration tests

### PR DESCRIPTION
**Pull request by Enriqe**
They are flaking on master

https://travis-ci.org/ampproject/amphtml/jobs/587116429
```
      IT => should send analytics event when navigating
        ✗ AssertionError: expected 'page-1' to equal 'page-2'
            at _callee2$ (home/travis/build/ampproject/amphtml/test/integration/test-amp-story-analytics.js:143:35)
            at tryCatch (home/travis/build/ampproject/amphtml/node_modules/regenerator-runtime/runtime.js:65:40)
            at Generator.invoke [as _invoke] (home/travis/build/ampproject/amphtml/node_modules/regenerator-runtime/runtime.js:303:22)
            at Generator.prototype.<computed> [as next] (home/travis/build/ampproject/amphtml/node_modules/regenerator-runtime/runtime.js:117:21)
            at asyncGeneratorStep (/tmp/637d6ae55d488af8c723bc9ffe1cea60.browserify.js:99961:103)
            at _next (/tmp/637d6ae55d488af8c723bc9ffe1cea60.browserify.js:99963:194)
```

https://travis-ci.org/ampproject/amphtml/jobs/587121381
```
DESCRIBE => amp-story analytics
  DESCRIBE => amp-story analytics
    DESCRIBE =>  
      IT => should send analytics event when entering bookend
        ✗ AssertionError: expected 'false' to equal 'true'
            at _callee3$ (home/travis/build/ampproject/amphtml/test/integration/test-amp-story-analytics.js:154:36)
            at tryCatch (home/travis/build/ampproject/amphtml/node_modules/regenerator-runtime/runtime.js:65:40)
            at Generator.invoke [as _invoke] (home/travis/build/ampproject/amphtml/node_modules/regenerator-runtime/runtime.js:303:22)
            at Generator.prototype.<computed> [as next] (home/travis/build/ampproject/amphtml/node_modules/regenerator-runtime/runtime.js:117:21)
            at asyncGeneratorStep (/tmp/d871b543b0ca58abfa25877e17f239cc.browserify.js:99961:103)
            at _next (/tmp/d871b543b0ca58abfa25877e17f239cc.browserify.js:99963:194)
```
/cc @choumx

### CODE REVIEW

**Comment by choumx** [comment]
Looks like flakiness is just on Chrome 77.0.3865 (Windows 10.0.0) FYI.

**Comment by choumx** [comment]
Do we have a "skip windows" test option?

**Comment by Enriqe** [comment]
That's true, unfortunately I don't think there is a "skip windows" test option

**Comment by choumx** [comment]
We can add one. ;)

**Comment by estherkim** [comment]
Wait no, I removed `skipWindows()` as a test config because we run almost all Sauce Labs tests on Windows. Skipping should be browser specific at most.

Also I was looking at these tests just now, I'm not sure the OS is the reason for the flakiness.

File: test/_init_tests.js [hunk 1]
```diff
@@ -148,6 +149,10 @@ class TestConfig {
     return this.skip(this.runOnIos);
   }
 
+  skipWindows() {
```
Comment (estherkim) [inline test/_init_tests.js]: :(

**Comment by Enriqe** [comment]
Alright I reverted the last commit which introduces the skip for windows. Let's just skip them 
altogether to unblock master and I'll take a closer look for the root cause.

**Comment by choumx** [comment]
Good point, what we want is to skip Chrome on Windows specifically. Either way.

**Pull request merged**